src/frontend/core/executor/impl/TriangleCountExecutor.cpp

```diff
@@ -477,43 +567,25 @@ long TriangleCountExecutor::getTriangleCount(int graphId, std::string host, int
                                                                                          masterIP,
                                                                                          adjustedAvailableFiles, threadPriority);
 
-                    std::vector<std::string> triangles = Utils::split(compositeTriangles, ':');
-                    std::vector<std::string>::iterator triangleIterator;
-
-                    triangleTreeMutex.lock();
-
-                    for (triangleIterator = triangles.begin(); triangleIterator != triangles.end(); ++triangleIterator) {
-                        std::string triangle = *triangleIterator;
-
-                        if (!triangle.empty() && triangle != "NILL") {
-                            std::vector<std::string> triangleVertexList = Utils::split(triangle, ',');
+                    triangleCount_logger.log("###COMPOSITE### Retrieved Composite triangle list ", "info");
```

Comment:
Rather than having these as info logs, can't we make them debug logs?

Reply:
Made it a debug log

Reply:
Ok